infra/aws/org-bootstrap.md

```diff
@@ -0,0 +1,394 @@
+  source  = "trussworks/iam-cross-acct-src/aws"
+  version = "1.0.0"
+
+  destination_account_ids = [
+    data.aws_caller_identity.current.account_id,
+    local.spacecats_infra_account_id
+  ]
+  destination_group_role  = "infra"
+}
+
+module "infra_group" {
+  source  = "trussworks/iam-user-group/aws"
+  version = "1.0.2"
+
+  user_list     = ["myuser"]
+  allowed_roles = [module.infra_group_role.arn]
+  group_name    = "infra"
+}
+
+resource "aws_iam_role_policy_attachment" "infra_local_policy_attachment" {
+  role       = module.infra_group_role.name
+  policy_arn = "arn:aws:iam::aws:policy/PowerUserAccess"
+}
+```
+
+Using that code, we get an IAM user `myuser` in the group `infra`, which
+has permission to assume the `infra` role in the current account and the
+`spacecats-infra` account. We also attach a policy to the `infra` role we
+created that grants power user access.
+
+All user creation and grouping should be done in the `id` account. In the
+the other accounts, we grant these groups permission to assume roles that
+grant whatever permissions they need. So in the `spacecats-infra` account,
+if we want to allow the `infra` group to have similar permissions, we can
+use this code in that account's namespace:
+
+```hcl
+locals {
+  spacecats_id_acct_id = <spacecats-id account number>
+
+module "infra_role" {
+  source  = "trussworks/iam-cross-acct-dest/aws"
+  version = "1.0.0"
+
+  iam_role_name             = "infra"
+  source_account_id         = local.spacecats_id_account_id
+}
+
+resource "aws_iam_role_policy_attachment" "infra_local_policy_attachment" {
+  role       = module.infra_role.iam_role_name
+  policy_arn = "arn:aws:iam::aws:policy/PowerUserAccess"
+}
+```
+
+This code creates a new `infra` role in the `spacecats-infra` account and
+allows users in the `spacecats-id` account to assume that role in this
+account as well. Note that we're not gating this based on the role in the
+`id` account; instead, we assume the `id` account will do that gating for
+us. Then we grant the `infra` role the same level of permissions those
+users had in the `id` account. Note that we *could* give them a different
+level of access if we wanted to -- general engineers will not have
+permission to do much of anything in the `id` account most of the time,
+but we could grant them power user access in a sandbox account, and
+perhaps only access to look at Cloudwatch logs in the staging or prod
+accounts. Using this method allows you to do all your user management in
+one place, and then give role-based access everywhere else.
+
+In order for these users to access other accounts, you'll need to set up
+their profiles for these other accounts like so:
+
+```text
+[profile spacecats-infra]
+source_profile = spacecats-id
+role_arn = arn:aws:iam::<spacecats-infra account number>:role/infra
+region = us-west-2
+output = json
+# If you have MFA turned on, you will also need to add the mfa_serial
+# line from the source_profile.
+```
+
+## Next Steps
```

Comment:
And additional next step could be to revisit the org-scp module README to enable SCPs like denying access to the root user in every account.

Reply:
Good call. I think there's some SCPs we can take from SABER and add to the org-scp module that might be useful too...I'll make a ticket for that.

Reply:
Added some language about this.